presentation/common/src/presentation-common/rules/content/ContentInstancesOfSpecificClassesSpecification.ts

```diff
@@ -27,6 +27,11 @@ export interface ContentInstancesOfSpecificClassesSpecification extends ContentS
    */
   classes: MultiSchemaClassesSpecification | MultiSchemaClassesSpecification[];
 
+  /**
+   * List of ECClass specifications whose ECInstances should be excluded.
+   */
+  excludedClasses?: MultiSchemaClassesSpecification | MultiSchemaClassesSpecification[];
+
   /** Whether to get content from instances of derived `classes`. */
   handleInstancesPolymorphically?: boolean;
```

Comment:
- I'd add this: <code>If set, acts as default value for `classes.arePolymorphic` and `excludedClasses.arePolymorphic`.</code>
- Should be marked `@deprecated`